packaging/fedora/snapd.spec

```diff
@@ -98,6 +99,7 @@ BuildRequires: golang(golang.org/x/crypto/ssh/terminal)
 BuildRequires: golang(golang.org/x/net/context)
 BuildRequires: golang(golang.org/x/net/context/ctxhttp)
 BuildRequires: golang(gopkg.in/check.v1)
+BuildRequires: golang(gopkg.in/cheggaaa/pb.v1)
```

Comment:
We could depend on it and then do some symlinks at the right spot to unblock the build and land it for now.

Reply:
Well, this `golang()` name doesn't exist yet at all...

Reply:
I implemented this, but I'm extremely unhappy that this situation even happened to begin with.